modules/nodes-cma-autoscaling-custom-prometheus-config.adoc

```diff
@@ -16,6 +16,7 @@ These steps are not required for an external Prometheus source.
 You must perform the following tasks, as described in this section:
```

Comment:
This is not really in scope of your PR but I was wondering if this section is really necessary since the second sentence above is "However, there are some additional configurations you must perform". I was just thinking because at first glance I read it like a prereq, but that was poor reading on my end.

Reply:
i'm not sure...i think longer procedures with a lot of repeating terms (service token, service account, create the role, add the role, etc.) like this one do benefit from a summary of the steps before getting into the steps but i agree it's probably not necessary or even that common to see!